app/models/item.rb

```diff
@@ -147,6 +149,12 @@ def score
 
   private
 
+  def valid_http_uri?(uri)
```

Comment:
The lines below are indented in the style used by the Rails team (an extra two spaces for `private`), do we want to do that here for consistency?